src/faostat_data_primap/download.py

```diff
@@ -241,17 +138,18 @@ def download_file(url_download: str, save_path: pathlib.Path) -> bool:
     -------
         True if the file was downloaded, False if a cached file was found
     """
-    if not save_path.exists():
-        with requests.get(url_download, stream=True, timeout=30) as response:
-            response.raise_for_status()
+    if save_path.exists():
+        if not save_path.is_symlink():
+            print(f"Skipping download of {save_path} because it already exists.")
+            return False
+        os.remove(save_path)
```

Comment:
This seems to solve the problem.
- Check if the file to download already exists (will return `True` for symlinks as well)
- Check if it's a symlink
	- if no, we have the file on disc (no need to download again)
	- if yes, it's not available on disc and datalad cannot unlock the file. In that case we need to delete the symlink before downloading